### PR TITLE
Display elapsed time for bazel query and content hash calculation

### PR DESCRIPTION
**Pull request by fa93hws**
In `generate-hashes` command, it takes a while to run `bazel query` and generate content hash for all source files in a large repo. It would be helpful if we know how long these steps take.

### CODE REVIEW

**Comment by fa93hws** [comment]
I've no idea about go error in https://github.com/Tinder/bazel-diff/runs/5835946773?check_suite_focus=true#step:4:1, it seems not to be relevant to my change...
Also this bazel error https://github.com/Tinder/bazel-diff/runs/5835946781?check_suite_focus=true#step:4:15 seems to be due to github.com/Tinder/bazel-diff/issues/106

File: src/main/java/com/bazel_diff/BazelClient.java [hunk 1]
```diff
@@ -37,25 +40,44 @@
...
     }
 
     @Override
     public List<BazelTarget> queryAllTargets() throws IOException {
+        Instant queryStartTime = Instant.now();
         List<Build.Target> targets = performBazelQuery("'//external:all-targets' + '//...:all-targets'");
+        Instant queryEndTime = Instant.now();
+        if (displayElapsedTime) {
+            long querySeconds = Duration.between(queryStartTime, queryEndTime).getSeconds();
+            System.out.printf("all targets queried in %d seconds%n", querySeconds);
```
Comment (tinder-maxwellelliott) [inline src/main/java/com/bazel_diff/BazelClient.java:62]:
```suggestion
            System.out.printf("BazelDiff: All targets queried in %d seconds%n", querySeconds);
```

File: src/main/java/com/bazel_diff/BazelClient.java [hunk 2]
```diff
@@ -37,25 +40,44 @@
...
-        return processBazelSourcefileTargets(performBazelQuery("kind('source file', //...:all-targets)"), true);
+        Instant queryStartTime = Instant.now();
+        List<Build.Target> targets = performBazelQuery("kind('source file', //...:all-targets)");
+        Instant queryEndTime = Instant.now();
+        Map<String, BazelSourceFileTarget> sourceFileTargets = processBazelSourcefileTargets(targets, true);
+        Instant contentHashEndTime = Instant.now();
+        if (displayElapsedTime) {
+            long querySeconds = Duration.between(queryStartTime, queryEndTime).getSeconds();
+            long contentHashSeconds = Duration.between(queryEndTime, contentHashEndTime).getSeconds();
+            System.out.printf("all source files queried in %d seconds%n", querySeconds);
```
Comment (tinder-maxwellelliott) [inline src/main/java/com/bazel_diff/BazelClient.java:77]:
```suggestion
            System.out.printf("BazelDiff: All source files queried in %d seconds%n", querySeconds);
```

File: src/main/java/com/bazel_diff/BazelClient.java [hunk 3]
```diff
@@ -37,25 +40,44 @@
...
+            System.out.printf("content hash calculated in %d seconds%n", contentHashSeconds);
```
Comment (tinder-maxwellelliott) [inline src/main/java/com/bazel_diff/BazelClient.java:78]:
```suggestion
            System.out.printf("BazelDiff: Content hash calculated in %d seconds%n", contentHashSeconds);
```

File: src/main/java/com/bazel_diff/main.java [hunk 4]
```diff
@@ -60,6 +67,11 @@ public Integer call() {
             FileWriter myWriter = new FileWriter(outputPath);
             myWriter.write(gson.toJson(hashes));
             myWriter.close();
+            Instant generateHashEndTime = Instant.now();
+            if (displayElapsedTime) {
+                long generateHashSeconds = Duration.between(generateHashStartTime, generateHashEndTime).getSeconds();
+                System.out.printf("generate-hashes command finishes in %d seconds%n", generateHashSeconds);
```
Comment (tinder-maxwellelliott) [inline src/main/java/com/bazel_diff/main.java:73]:
```suggestion
                System.out.printf("BazelDiff: Generate-hashes command finishes in %d seconds%n", generateHashSeconds);
```

**Review by tinder-maxwellelliott — APPROVED** [review]

**Pull request merged**